packages/utils/time.ts

```diff
@@ -96,7 +96,7 @@ export const formatMsToRelative = (ms: number) => {
 	// The expected pattern for invalid date formatting in JS is to return the string "Invalid
 	// Date", so we do that here. If we don't, dayjs will process the invalid date and return "a
 	// month ago", somehow...
-	if (!d.isValid()) return 'Invalid Date';
+	if (!d.isValid()) return 'Invalid date';
```

Comment:
Related to [this test failure](https://github.com/laurent22/joplin/actions/runs/9601789533/job/26481119058#step:14:1454):
```
 ➤ YN0000: [@joplin/lib]: FAIL services/noteList/renderViewProps.test.js
➤ YN0000: [@joplin/lib]:   ● renderViewProps › should handle invalid view props
➤ YN0000: [@joplin/lib]: 
➤ YN0000: [@joplin/lib]:     expect(received).toEqual(expected) // deep equality
➤ YN0000: [@joplin/lib]: 
➤ YN0000: [@joplin/lib]:     - Expected  - 1
➤ YN0000: [@joplin/lib]:     + Received  + 1
➤ YN0000: [@joplin/lib]: 
➤ YN0000: [@joplin/lib]:       Object {
➤ YN0000: [@joplin/lib]:         "note": Object {
➤ YN0000: [@joplin/lib]:           "tags": 123,
➤ YN0000: [@joplin/lib]:     -     "user_updated_time": "Invalid date",
➤ YN0000: [@joplin/lib]:     +     "user_updated_time": "Invalid Date",
➤ YN0000: [@joplin/lib]:         },
➤ YN0000: [@joplin/lib]:       }
➤ YN0000: [@joplin/lib]: 
➤ YN0000: [@joplin/lib]:       50 | 		Logger.globalLogger.enabled = true;
➤ YN0000: [@joplin/lib]:       51 |
➤ YN0000: [@joplin/lib]:     > 52 | 		expect(view).toEqual({
➤ YN0000: [@joplin/lib]:          | 		             ^
➤ YN0000: [@joplin/lib]:       53 | 			note: {
➤ YN0000: [@joplin/lib]:       54 | 				user_updated_time: 'Invalid date',
➤ YN0000: [@joplin/lib]:       55 | 				tags: 123,
➤ YN0000: [@joplin/lib]: 
➤ YN0000: [@joplin/lib]:       at toEqual (services/noteList/renderViewProps.test.ts:52:16)
➤ YN0000: [@joplin/lib]:       at fulfilled (services/noteList/renderViewProps.test.js:12:24)
➤ YN0000: [@joplin/lib]: 
```